Platform.Data.Triplets.Kernel/Common.h

```diff
@@ -54,14 +54,27 @@ typedef unsigned_integer link_index; // Short for links' array index, unsigned i
 #endif
 
 #define SUCCESS_RESULT 1
-#define succeeded(x) SUCCESS_RESULT == (x)
+#define succeeded(x) (SUCCESS_RESULT == (x))
 #define ERROR_RESULT 0
-#define failed(x) SUCCESS_RESULT != (x)
+#define failed(x) (SUCCESS_RESULT != (x))
 
-signed_integer Error(char* message);
+#endif
```

Comment:
What is this endif for?)
Good practise is marking them:
```c
#ifdef DEFINE_1
#ifdef DEFINE_2
/* some code... */
#endif // DEFINE_1
#endif // DEFINE_2
```

Reply:
Thanks, good advice